server.go

```diff
@@ -324,190 +339,75 @@ func (s *Server) handlerAddMovie(w http.ResponseWriter, r *http.Request) {
 			s.l.Error("Error parsing movie form: %v", err)
 		}
 
-		errText := []string{}
+		// errText := []string{}
 
-		// Get the links from the corresponding input field
-		linktext := strings.ReplaceAll(r.FormValue("Links"), "\r", "")
-		data.ValLinks = linktext
+		movie := &common.Movie{}
 
-		maxLinkLength, err := s.data.GetCfgInt(ConfigMaxLinkLength, DefaultMaxLinkLength)
-		if err != nil {
-			s.l.Error("Unable to get %q: %v", ConfigMaxTitleLength, err)
-			s.doError(
-				http.StatusInternalServerError,
-				"something went wrong :C",
-				w, r)
-			return
-		}
-
-		if len(linktext) > maxLinkLength {
-			s.l.Debug("Links too long: %d", len(linktext))
-			data.ErrTitle = true
-			errText = append(errText, "Links too long!")
-		}
-
-		links := strings.Split(linktext, "\n")
-		links, err = common.VerifyLinks(links)
-		if err != nil {
-			s.l.Error("bad link: %v", err)
-			data.ErrLinks = true
-			errText = append(errText, "Invalid link(s) given.")
-		}
-
-		remarks := strings.ReplaceAll(r.FormValue("Remarks"), "\r", "")
-		data.ValRemarks = remarks
-
-		maxRemarksLength, err := s.data.GetCfgInt(ConfigMaxRemarksLength, DefaultMaxRemarksLength)
-		if err != nil {
-			s.l.Error("Unable to get %q: %v", ConfigMaxRemarksLength, err)
-			s.doError(
-				http.StatusInternalServerError,
-				"something went wrong :C",
-				w, r)
-			return
-		}
-
-		if len(remarks) > maxRemarksLength {
-			s.l.Debug("Remarks too long: %d", len(remarks))
-			data.ErrRemarks = true
-			errText = append(errText, "Remarks too long!")
-		}
+		if autofillEnabled {
```

Comment:
This could prove to be problematic since autofill disabled and autofillbox on will end up in the everything is disabled block.

Reply:
fixed in 31252dc